ui/src/constants.js

```diff
@@ -23,6 +23,7 @@ export const STATUS_FAILED = 'Failed';
 export const STATUS_AVAILABLE = 'Available';
 export const STATUS_RELEASED = 'Released';
 export const STATUS_UNKNOWN = 'Unknown';
+export const STATUS_READY = 'Ready';
```

Comment:
Should explain in the commit message **WHY** we change from `Available` to `Ready` (you've have done it in the PR description, but not in the commit message).